view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js

```diff
@@ -460,7 +457,7 @@ define(
                     mount(actionNode);
                 }
             },
-            handleOnSubmit: function(state, component) {
+            handleOnSubmit: async function(state, component) {
```

Comment:
Is this method different from the one we use at line `125`? If it's the same, we might need an `await` there too.